docs/api/trips.md

```
@@ -0,0 +1,518 @@
+  }
+]
+```
+
+### Errors
+
+- `401` - Not authenticated
+- `429` - Rate limit exceeded
+
+### Examples
+
+**cURL**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)